cmd/server/main.go

```diff
@@ -0,0 +1,36 @@
+package main
+
+import (
+	"log"
+
+	// _ "github.com/go-sql-driver/mysql"
+	rmx "github.com/rog-golang-buddies/rapidmidiex"
+)
+
+func main() {
+	log.Println("loading config")
+	err := rmx.LoadConfig()
+	if err != nil {
+		log.Fatalf("fatal error config file: %v", err.Error())
+	}
+
+	// database connection shouldn't be implemented here
```

Comment:
Go ahead and delete this commented-out code if you're going to use `config.go` to setup the DB connection